docs/symfony/get-it-started.md

```diff
@@ -22,7 +22,7 @@ _**Note**: Where payum/offline is a php payum extension, you can for example cha
 
 _**Note**: Use payum/payum if you want to install all gateways at once._
 
-Enable the bundle in the kernel:
+Enable the bundle in the kernel (only necessary for Symfony <=3):
```

Comment:
Symfony 3 isn't supported anymore in PayumBundle, so I think all references and config for Symfony 3 can be removed